api/envoy/config/bootstrap/v4alpha/bootstrap.proto

```diff
@@ -96,9 +96,9 @@ message Bootstrap {
     core.v4alpha.ApiConfigSource ads_config = 3;
   }
 
-  reserved 10, 11, 8, 9;
+  reserved 10, 11, 8, 9, 20;
```

Comment:
Why is 20 a reserved field?  Is it standard practice to reserve the ID of deprecated fields?

Reply:
I also didn't notice this before. Now I realize what "marking deprecated" in v3 results in.